spec/mongoid/touchable_spec.rb

```diff
@@ -584,5 +584,118 @@
         end
       end
     end
+
+    context "when the touch option is false" do
+
+      shared_examples "does not update the parent" do
+
+        let!(:start_time) { Timecop.freeze(Time.at(Time.now.to_i)) }
+
+        let(:update_time) do
+          Timecop.freeze(Time.at(Time.now.to_i) + 2)
+        end
+
+        after do
+          Timecop.return
+        end
```

Comment:
In my version I removed Timecop, because I wanted to make sure various timestamps are actually synchronized when touching. Timecop (because it synchronizes time) will lead to false positives.

Reply:
Can you explain the case specifically where timecop will lead to false positives?